ci/env.sh

```diff
@@ -9,3 +9,4 @@ export AWS_DEFAULT_REGION=us-east-1
 export AWS_REGION=us-east-1
 export RACK_NAME=$(cat /tmp/convox-rack-name)
 export VERSION=${VERSION:-${CIRCLE_TAG}}
+export LATEST=$(curl -s https://api.github.com/repos/convox/rack/releases/latest | grep tag_name | cut -f 2 -d : | tr -dc '0-9')
```

Comment:
`tag_name`, shouldn't it be `name`? since we are calling latest release api,
also `jq` can make it simple
```
curl -s https://api.github.com/repos/convox/rack/releases/latest | jq -r '.name'
```

Reply:
`tag_name` is in the response body of github api

Reply:
@nightfury1204 a release `name` can be changed and we use the `tag_name` to build the container images (used by the ApiWebService and ApiMonitorService)

Reply:
got it thanks